low-level/sensors/impl/generic/iem_sensor.py

```diff
@@ -249,6 +251,12 @@ def _send_msg(self, iem_components, log_timestamp):
         if not self._are_components_in_range(**args):
             return
 
+        # component-id for sspl=005
+        if component_id == "005":
+            event_code = component_id + module_id + event_id
+            impact = Iem().EVENT_STRING[event_code][1]
```

Comment:
Impact & recommendations needs to recommended for other component IEM alerts too. Will discuss in relevant forums.